rct229/rulesets/ashrae9012019/section19/section19rule15.py

```diff
@@ -0,0 +1,213 @@
+                },
+            )
+
+        def is_applicable(self, context, data=None):
+            hvac_b = context.baseline
+            hvac_id_b = hvac_b["id"]
+            baseline_system_types_dict = data["baseline_system_types_dict"]
+
+            return any(
+                hvac_id_b in baseline_system_types_dict[system_type]
+                for system_type in baseline_system_types_dict.keys()
+            )
+
+        def get_calc_vals(self, context, data=None):
+            hvac_b = context.baseline
+            hvac_id_b = hvac_b["id"]
+
+            fan_system_b = hvac_b["fan_system"]
+
+            fan_info_b = (
+                get_fan_system_object_supply_return_exhaust_relief_total_power_flow(
+                    fan_system_b
+                )
+            )
+
+            supply_fan_qty_b = fan_info_b["supply_fans_qty"]
+            supply_fan_airflow_b = fan_info_b["supply_fans_airflow"]
+
+            minimum_outdoor_airflow_b = fan_system_b["minimum_outdoor_airflow"]
+
+            return {
+                "hvac_id_b": hvac_id_b,
+                "supply_fan_qty_b": supply_fan_qty_b,
+                "supply_fan_airflow_b": supply_fan_airflow_b,
+                "minimum_outdoor_airflow_b": minimum_outdoor_airflow_b,
+            }
+
+        def manual_check_required(self, context, calc_vals=None, data=None):
+            hvac_id_b = calc_vals["hvac_id_b"]
+            supply_fan_qty_b = calc_vals["supply_fan_qty_b"]
+            supply_fan_airflow_b = calc_vals["supply_fan_airflow_b"]
+            hvac_info_dict_b = data["hvac_info_dict_b"][hvac_id_b]
+
+            return (
+                supply_fan_qty_b == 1
+                and not hvac_info_dict_b["all_design_setpoints_105"]
+                and std_equal(
+                    hvac_info_dict_b["proposed_supply_cfm"],
+                    supply_fan_airflow_b,
+                )
+            ) and (supply_fan_qty_b != 1)
+
+        def get_manual_check_required_msg(self, context, calc_vals=None, data=None):
+            hvac_id_b = calc_vals["hvac_id_b"]
+            supply_fan_qty_b = calc_vals["supply_fan_qty_b"]
+            supply_fan_airflow_b = calc_vals["supply_fan_airflow_b"]
+            hvac_info_dict_b = data["hvac_info_dict_b"]["hvac_id_b"]
+
+            if (
+                supply_fan_qty_b == 1
+                and not hvac_info_dict_b["all_design_setpoints_105"]
+                and std_equal(
+                    hvac_info_dict_b["proposed_supply_cfm"],
+                    supply_fan_airflow_b,
+                )
+            ):
+                undetermined_msg = f"{hvac_id_b} was not modeled with a supply air temperature set point of 105°F. The baseline and proposed supply cfm was modeled identically at {hvac_info_dict_b['proposed_supply_cfm']} CFM. Manual review is required to determine if the airflow rate was modeled to comply with applicable codes or accreditation standards. If not, fail."
+            elif supply_fan_qty_b != 1:
```

Comment:
It is one or another, so in here you can safely use `else`

To make the code cleaner, you can also set to 
```python
if supply_fan_qty_b != 1: message...
else: message...
```
This is recommendation, not required, I will leave it to you to decide.

Reply:
I used ```else``` because this makes the code more succinct.